modules/help_editor/php/help_editor.class.inc

```diff
@@ -99,66 +99,24 @@ class Help_Editor extends \NDB_Menu_Filter
         $this->addBasicText('topic', 'Help topic:');
         $this->addBasicText('keyword', 'Search keyword');
```

Comment:
everythings looks great! i have a minor request for change. it looks funny on the front-end
```suggestion
        $this->addBasicText('keyword', 'Search keyword:');
```

<img width="891" alt="screenshot 2018-10-22 10 33 46" src="https://user-images.githubusercontent.com/17415878/47298574-25501600-d5e6-11e8-9a9d-defbff404c0b.png">

ready to approve after this!